src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -941,6 +935,12 @@ private Button createNewEntryFromIdButton() {
         return newEntryFromIdButton;
     }
 
+    private CheckMenuItem createSidePaneCheckMenuItem(SidePaneContainerView container, ActionFactory factory, SidePaneType sidePane) {
+        CheckMenuItem checkMenuItem = factory.createCheckMenuItem(sidePane.getToggleAction(), container.getToggleCommandFor(sidePane), container.sidePaneVisibleProperty(sidePane).get());
+        EasyBind.subscribe(container.sidePaneVisibleProperty(sidePane), checkMenuItem::setSelected);
+        return checkMenuItem;
+    }
```

Comment:
This could probably be part of the ActionFactory itself overloading the present createCheckMenuItem with the binding value?